action.yml

```diff
@@ -14,6 +14,10 @@ inputs:
     description: 'Check for changes using only this list of files (Defaults to the entire repo)'
     required: false
     default: ""
+  sha:
+    description: 'Commit sha'
```

Comment:
Can we also make the description more verbose and use a default.

```yaml
description: 'Specify a different commit SHA to be used for comparing changes'
required: true
default: ${{ github.sha }}
```

Reply:
Done